tests/basic-test.sh

```diff
@@ -41,6 +41,7 @@ fi
 validate_checkout_basic() {
     (cd $1;
      assert_has_file firstfile
+     assert_not_streq $(stat -c '%h' firstfile) 1
```

Comment:
Yeah, this only works in bare right now.  This whole suite really needs rewriting.